src/kibana/components/filter_manager/lib/range.js

```diff
@@ -1,14 +1,32 @@
 define(function (require) {
   var _ = require('lodash');
+  var OPERANDS_IN_RANGE = 2;
+
   return function buildRangeFilter(field, params, indexPattern, formattedValue) {
     var filter = { meta: { index: indexPattern.id } };
     if (formattedValue) filter.meta.formattedValue = formattedValue;
-    params = _.omit(params, _.isFunction);
 
-    if (params.gte && params.gt) throw new Error('gte and gt are mutually exclusive');
-    if (params.lte && params.lt) throw new Error('lte and lt are mutually exclusive');
+    params = _.clone(params);
```

Comment:
That _.omit is important - there's a method attached to `params` elsewhere that needs to be removed for filters to be generated correctly